mxcube3/routes/qutils.py

```diff
@@ -1081,15 +1079,18 @@ def init_signals(queue):
     """
     import signals
 
-    for signal in signals.collect_signals:
-        mxcube.collect.connect(mxcube.collect, signal,
-                               signals.task_event_callback)
```

Comment:
With this change, task_signals including "collect_finished" and "warning" are not connected to any event and will not be handled.